packages/coverage-c8/src/provider.ts

```diff
@@ -102,6 +102,12 @@ export class C8CoverageProvider implements CoverageProvider {
 
     await report.run()
     await checkCoverages(this.options, report)
+
+    // Note that this will only clean up the V8 reports generated so far.
```

Comment:
More details about this here: https://github.com/vitest-dev/vitest/issues/2108#issuecomment-1276557168